### ISSUE TITLE
Oni2 can't handle accents

### ISSUE THREAD

**Issue by dsejas** [onivim/oni2#787]
**Description:** When trying to write a word with an accent, Oni does not display the accented letter.
**Version:** #53dfb8be, dated September 22nd, 2019
**How to reproduce:** In INSERT mode, type the Spanish word "qué" (with an accent over the "e"). Oni will display "qu".

**Comment by szbergeron** [comment]
AFAIK this is currently blocked by https://github.com/revery-ui/revery/pull/555, as SDL will bring in ime support. Backspacing through unicode is also something that needs to be done differently. It might make sense to make a tracker for international input in general to track all this stuff.

**Comment by johnridesabike** [comment]
Is this also why emojis don't work, or would that be a separate issue?

**Comment by chriskrycho** [comment]
I *suspect* this is the same issue I just hit: you can't currently enter a curly quote Unicode character `’`. The normal keystroke on macOS, <kbd>⌘</kbd><kbd>⇧</kbd><kbd>]</kbd>, results in this output:

> 

Sublime displays this in grayed-out text (presumably b/c it can't properly display it) as `<0x80><0x99>`, and VS Code displays it like this:

<img width="27" alt="Screen Shot 2020-03-03 at 19 53 01" src="https://user-images.githubusercontent.com/2403023/75840491-9847d000-5d88-11ea-940d-686dcbb90375.png">

**Comment by baransu** [comment]
Still an issue with characters like `ńąćśźż` etc.

**Comment by bryphe** [comment]
This should be fixed now with this PR:
https://github.com/onivim/oni2/pull/1608

![image](https://user-images.githubusercontent.com/13532591/80773574-23d2a880-8b0f-11ea-8592-676e8b6f33e9.png)

(with the caveat that we still don't have _font fallback_ implemented - tracking that as part of the CJK work here: https://github.com/onivim/oni2/issues/868 - that's also the blocker for emojis for us!)

**Comment by chriskrycho** [comment]
Awesome! Thank you. Still enjoying Oni – not quite a daily driver, but using it pretty much every day!
